### ISSUE TITLE
Ollama

### ISSUE THREAD

**Issue by nsosio** [premAI-io/benchmarks#43]
see [here](https://github.com/jmorganca/ollama)

**Comment by nsosio** [comment]
@Anindyadeep can you check if they support llama2/mistral? Otherwise let's close the issue

**Comment by Anindyadeep** [comment]
Yes we need to put this in medium priority, since it is an engine

**Comment by Anindyadeep** [comment]
> @Anindyadeep can you check if they support llama2/mistral? Otherwise let's close the issue

Yes we need to put this in medium priority, since it is an engine

**Comment by Interpause** [comment]
isn't ollama practically a wrapper around llama.cpp though?
